src/Fractal.php

```diff
@@ -238,6 +241,27 @@ public function parseExcludes($excludes)
     }
 
     /**
+     * Specify the fieldsets to include in the response.
+     *
+     * @param array $fieldsets array with key = resourceName and value = fields to include
+     *                                (array or comma separated string with field names)
+     *
+     * @return $this
+     */
+    public function parseFieldsets(array $fieldsets)
+    {
+        foreach ($fieldsets as $key => $fields) {
```

Comment:
I value good looking code higher then having ticked all the micro optimization checkboxes 😄    

But leave this as it is for now, I'll play around a bit with it myself to find out what feels best to me.